lunatrace/bsl/frontend/src/pages/project/builds/details/vulnerable-packages/body/DepChains.tsx

```diff
@@ -14,8 +14,11 @@
 import React, { useState } from 'react';
 import { Badge, NavLink } from 'react-bootstrap';
 import { ChevronRight, ChevronsRight, Maximize2, Minimize2 } from 'react-feather';
+import PerfectScrollbar from 'react-perfect-scrollbar';
```

Comment:
What does this library do that can't be accomplished with css overflow property?

Reply:
oh my god https://perfectscrollbar.com/

Reply:
great touch support, animations like bounce back, pretty CSS for the bars, controls over the bar size, etc. It works well and its two lines.